signadot/operator/values.yaml

```diff
@@ -51,27 +51,44 @@
 # - ns1
 # - ns2
 
-# Image customization parameters:
-#
-# operator:
-#   image: my-private-registry/signadot/operator:latest
-#   imagePullPolicy: Always
- 
-# agent:
-#   image: my-private-registry/signadot/agent:latest
-#   imagePullPolicy: Always
-
-# routeServer:
-#   image: my-private-registry/signadot/route-server:latest
-#   imagePullPolicy: Always
 
-# ioContextServer:
-#   image: my-private-registry/signadot/io-context-server:latest
-#   imagePullPolicy: Always
-
-# kubeRBACProxy:
-#   image: my-private-registry/gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
-#   imagePullPolicy: Always
+operator:
```

Comment:
Do we want to target having helm values for sidecars and init images as well, or is that too much for the v0.19.2 timeline?

Reply:
I was treating that as too much for the v0.19.2 timeline (we don't even have a way to configure this in the app), but I'm open to reconsider it (in any case, I would do it in a separate PR given it requires app changes too).

Reply:
maybe opening an issue would help track it.  I agree it is not needed for v0.19.2

Reply:
I created https://github.com/signadot/signadot/issues/5254 for that.